### DISCUSSION TITLE
Failures for awaited assertions are marked by Allure as Broken, rather than a Failure

### DISCUSSION THREAD

**Discussion by Maximus1285** [allure-framework/allure2#2356] [Bug Reports] [answered]
Using [allure-playwright](https://www.npmjs.com/package/allure-playwright#hooks-and-api-calls)

Having awaited assertions like this:
`await expect(page.getByTestId('locator')).toBeEnabled();`

Playwright will attempt this expectation every certain millisec util it meets the expected behavior or it times out trying, in which case it would be considered by Playwright as a failed test. If if times out, but Allure treats it rather as a test defect and marks it as Broken.

I think this is a bug on how test results are interpreted for this kind of expectations and the package doesn't expose any api to override the criteria to categorize the different test results according to each scenario

**Comment by epszaw** [comment]
Hey, @Maximus1285! Usually, we mark tests as broken when something went wrong outside of test context (runtime errors, etc.). You're right, that [we set `BROKEN` status for timed out tests](https://github.com/allure-framework/allure-js/blob/main/packages/allure-playwright/src/index.ts#L388), but I need some time to figure out, why we handle this in this way.

**Reply by Maximus1285** [reply, marked as answer]
@epszaw I found what the issue was. I figured both global timeout and expect timeout had the same value, so it was always defaulting to the global timeout. 

I've decreased the expect timeout and now tests are successfully being marked as `Failed`
I think we can then close this issue

**Reply by epszaw** [reply]
Good job here :)
